src/connection/WalletConnectV2.ts

```diff
@@ -61,3 +61,37 @@ export class WalletConnectV2 extends WalletConnect {
     return super.activate(chainId)
   }
 }
+
+// Custom class for Uniswap Wallet specific functionality
+export class UniwalletConnect extends WalletConnectV2 {
```

Comment:
since this is called UniswapWalletV2 everywhere else should you just add v2 to the naming here